fixed-requirements.txt

```diff
@@ -5,7 +5,7 @@ greenlet==0.4.15
 # depend on rely
 eventlet==0.24.1
 gunicorn==19.9.0
-kombu==4.2.2
+kombu==4.2.2.post1
```

Comment:
holy

Reply:
Yeah, it's part of a Python versioning standard - https://www.python.org/dev/peps/pep-0440/#post-releases

I'm fine with it since it's considered a "final" and not a dev / beta release. But yeah, I would definitely prefer them to actually bump a version to 4.2.3 or similar.